lib/js/components/Pagination/Pagination.vue

```diff
@@ -0,0 +1,369 @@
+<template>
+	<div
+		:class="{
+			pagination: true,
+			'-compact': layout === PAGINATION_LAYOUTS.COMPACT,
+			'-centered': isCentered,
```

Comment:
This class shouldn't be set when accessory slot is filled: https://miro.com/app/board/uXjVPVmlbI8=/?moveToWidget=3458764538534562880&cot=14